ClosedXML/Utils/OpenXmlHelper.cs

```diff
@@ -81,6 +82,52 @@ public static XLColor ToClosedXMLColor(this X14.ColorType openXMLColor, IDiction
 
         #endregion Public Methods
 
+        #region Internal Methods
+        /// <summary>
+        /// Find image position in drawing.
+        /// </summary>
+        /// <param name="stream">elsx stream</param>
+        /// <param name="sheetName">sheet name</param>
+        /// <param name="relId">picture relId</param>
+        /// <returns>position index in drawing part</returns>
+        internal static int FindImageRelIdDrawingOrder(System.IO.Stream stream, string sheetName, string relId)
```

Comment:
This method shouldn't be in the ClosedXML project, because it is only used for tests. Tests also shouldn't deal with xml structure directly, unless there is a good reason, see comment near the test.